### PR TITLE
Block vote4

### PR DESCRIPTION
**Pull request by bvdh**
Changes as agreed in Block-Vote-4

### CODE REVIEW

File: input/pagecontent/3-2-0-infrastructureevents.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 Supporting events used in basic FHIRcast interactions.
```
Comment (EricOnFHIR) [inline input/pagecontent/3-2-0-infrastructureevents.md:1]: I agree with these changes

File: package-list.json [hunk 2]
```diff
@@ -7,10 +7,10 @@
       {
         "version": "current",
         "desc": "Continuous Integration Build (latest in version control)",
-        "path": "http://hl7.org/fhir/uv/fhircast",
+        "path": "http://hl7.org/fhir/uv/fhircast/build",
         "status": "ci-build",
         "sequence": "STU3",
-        "fhirversion": "4.0.1",
+        "fhirversion": "4.3.0",
```
Comment (EricOnFHIR) [inline package-list.json:13]: Would this now be "5.0.0"?

**Review by EricOnFHIR — CHANGES_REQUESTED** [review]
I apologize for the event name misunderstanding - I know it was significant editing work.

File: input/pagecontent/2-4-Subscribing.md [hunk 3]
```diff
@@ -50,7 +52,7 @@ hub.channel.type=websocket&hub.mode=subscribe&hub.topic=fdb2f928-5546-4f52-87a0-
 ### Subscription Response
 
 Upon receiving subscription or unsubscription requests, the Hub SHALL respond to a subscription request with an HTTP 202 "Accepted" response. This indicates that the request was received and will now be verified by the Hub.
-
+https://github.com/HL7/fhircast-docs/tree/blockVote4
```
Comment (isaacvetter) [inline input/pagecontent/2-4-Subscribing.md:55]:
```suggestion

```

Is this url a copy-paste error?

Comment (isaacvetter) [inline input/pagecontent/2-4-Subscribing.md:55]:
```suggestion

```

Is this url a copy-paste error?

File: package-list.json (hunk 2 above)
Comment (isaacvetter) [reply package-list.json:13]: Has anyone ever tested R4B resources with FHIRcast? Why shouldn't we keep this at 4? (Like until folks begin supporting R6?)

File: input/pagecontent/2-4-Subscribing.md [hunk 4]
```diff
@@ -67,7 +69,7 @@ HTTP/1.1 202 Accepted
 
 ### Subscription Confirmation
 
-To confirm a subscription request, upon the Subscriber establishing a WebSocket connection to the `hub.channel.endpoint` WSS URL, the Hub SHALL send a confirmation. This confirmation includes the following elements:
+To confirm a subscription request, upon the subscriber establishing a WebSocket connection to the `hub.channel.endpoint` WSS URL, the Hub SHALL send a confirmation over the websocket channel. This confirmation includes the following elements:
```
[the remark this answers is not in the document]
Comment (bvdh) [reply input/pagecontent/2-4-Subscribing.md:72]: Fixed

File: package-list.json (hunk 2 above)
Comment (bvdh) [reply package-list.json:13]: 4.3.0 is required for SubscriptionTopic.

File: input/pagecontent/2-4-Subscribing.md (hunk 3 above)
Comment (bvdh) [reply input/pagecontent/2-4-Subscribing.md:55]: yes, fixed

**Comment by bvdh** [comment]
I've updated the pull request with the revised resolution on  FHIR-37074.
There is still an issue with compiling SubscriptionTopic definitions, but that should not prevent a review.

File: input/pagecontent/2-3-Events.md [hunk 5]
```diff
@@ -35,16 +35,17 @@ EventName ::= (FHIRresource | '*') ('-') ( 'open' | 'close' | 'update' | 'select
...
 
-Event names are unique and case-insensitive. Implementers may define their own events. Such proprietary events SHALL be named with reverse domain notation (e.g. `org.example.patient_transmogrify`). Reverse domain notation SHALL NOT be used by a standard event catalog. Proprietary events SHALL NOT contain a dash ("-").
+Event names are unique and case-insensitive. It is RECOMMENDED to use [Upper-Camel](https://en.wikipedia.org/wiki/Camel_case) case.
 
-When subscribing to FHIRcast events a list of events is added. These events may contain wild cards. Wild cards are expressed as a `*` replacing either the `FHIRresource` or `suffix`  with `*` indicates any events that match the resulting definition are requested. The event `*` means the subscriber subscribes to any event. The table below shows some typical examples.
+Implementers may define their own events. Such proprietary events SHALL be named with reverse domain notation (e.g. `org.example.patient_transmogrify`). Reverse domain notation SHALL NOT be used by a standard event catalog. Proprietary events SHALL NOT contain a dash ("-").
```
Comment (EricOnFHIR) [inline input/pagecontent/2-3-Events.md:40]: Will the reverse domain notation conflict with the -forward proposal which should be coming from Isaac?  I guess we can adjudicate then.

File: input/pagecontent/3-2-2-Heartbeat.md [hunk 6]
```diff
@@ -1,16 +1,16 @@
-{% include questionnote.html text='Implementer feedback is requested for the need to support heartbeats for WebSockets.' %}
+{% include questionnote.html text='Implementer feedback is requested for the need to support Heartbeat.htmls for WebSockets.' %}
```
Comment (EricOnFHIR) [inline input/pagecontent/3-2-2-Heartbeat.md:1]: Do you mean to use Heartbeat.htmls or Heartbeat events?  My guess is a copy/paste thing assuming that it was going to be in a link.  Couple of places in this file and other infrastructure events.

File: input/pagecontent/3-2-3-UserLogout.md [hunk 7]
```diff
@@ -1,16 +1,16 @@
-### Event-name: userlogout
+### Event-name: UserLogout.html
```
Comment (EricOnFHIR) [inline input/pagecontent/3-2-3-UserLogout.md:1]: Same comment as Heartbeat.html, do you mean UserLogout event?

File: input/pagecontent/3-2-4-UserHibernate.md [hunk 8]
```diff
@@ -1,12 +1,12 @@
-### Event-name: userhibernate
+### Event-name: UserHibernate.html
```
Comment (EricOnFHIR) [inline input/pagecontent/3-2-4-UserHibernate.md:1]: Drop the .html?

**Review by EricOnFHIR — APPROVED** [review]

**Pull request merged**